src/connections/stream_api.rs

```diff
@@ -209,6 +210,11 @@ impl<State> ConnectedStreamApi<State> {
             id: generate_rand_id(),
             want_ack,
             channel: channel.channel(),
+            public_key: Vec::new(),
+            pki_encrypted: false,
+            next_hop: 0,
+            relay_node: 0,
+            tx_after: 0,
```

Comment:
Does `Default::default()` work?
```suggestion
            ..Default::default()
```

Reply:
yup, done